src/result.rs

```diff
@@ -75,6 +75,8 @@ pub enum EvalAltResult {
     ErrorDotExpr(String, Position),
     /// Arithmetic error encountered. Wrapped value is the error message.
     ErrorArithmetic(String, Position),
+    /// Parameter was incorrectly typed.
+    ErrorFnInputType(String, Position),
```

Comment:
I wonder where this is going to hit... parameters are never supposed to be of the incorrect type...

Reply:
That was a left-over from my varadic stuff. I'll remove it.